webui/src/pages/repositories/index.jsx

```diff
@@ -219,14 +229,23 @@ const RepositoriesPage = () => {
                     />
 
                 <CreateRepositoryModal
-                    onCancel={() => setShowCreateRepositoryModal(false)}
+                    onCancel={() => {
+                        setShowCreateRepositoryModal(false);
+                        setCreateRepoError(null);
+                    }}
                     show={showCreateRepositoryModal}
-                    error={createError}
-                    onSubmit={(repo) => createRepo(repo)}/>
+                    error={createRepoError}
+                    onSubmit={async (repo) => await createRepo(repo, true)}/>
```

Comment:
Isn't this identical to
```suggestion
                    onSubmit={(repo) => createRepo(repo, true)}/>
```
?

Both Promise to return the result of `createRepo(repo, true)` or fail.

Reply:
true